### PR TITLE
17 the 8 white cards are visible to the user

### PR DESCRIPTION
**Pull request by anhuwi**
play white cards:
- you can choose white cards on a black card
- a play gets post
There ar emore than 8 cards??

### CODE REVIEW

File: src/components/views/PlayWhites.js [hunk 1]
```diff
@@ -4,52 +4,144 @@ import BaseContainer from "components/ui/BaseContainer";
...
+  const userId = localStorage.getItem("id")
+  const token = localStorage.getItem("token")
+
+    const WhiteCard = ({card}) => {
+        // use react-router-dom's hook to access the history
+        const history = useHistory();
+
+        // put the black Card to the Server and proceed to main menu
+        const selectCard = async ()  => {
+            // TODO: onClick: ask to confirm this card
```
Comment (JoMuelle) [inline src/components/views/PlayWhites.js:29]: use window.confirm(), it is also used in Edit User Page Line 150.

File: src/components/views/PlayWhites.js [hunk 2]
```diff
@@ -4,52 +4,144 @@ import BaseContainer from "components/ui/BaseContainer";
...
-          // TODO: onClick: ask to confirm this card, then make api call to submit this card choice, then reload page
-          // onClick={() => history.push('/game/playWhites')}
-          children={"This is a white card"}
-        />
-
-
-      </MenuContainer>
+        <Header view="game"/>
+        <div className={"game description"}>
+            <h1>Choose a White Card for the provided Black Card</h1>
```
Comment (JoMuelle) [inline src/components/views/PlayWhites.js:128]: Change the text to "Use a White Card to fill in the Blank"

File: src/components/views/PlayWhites.js [hunk 3]
```diff
@@ -4,52 +4,144 @@ import BaseContainer from "components/ui/BaseContainer";
...
+        </div>
+        <div>
+            <button onClick = {() => setCount(count + 1)}>
+                Or Skip this Card </button>
```
Comment (JoMuelle) [inline src/components/views/PlayWhites.js:132]: Do we already allow for skipping cards? And what is the cost for skipping cards?
- if we allow for skipping users will likely only play on Black cards that perfectly suits their white cards which ruins the point of the game.

**Review by JoMuelle — CHANGES_REQUESTED** [review]
Looks good, few minor changes. 
I think we shouldn't allow (yet) for skipping cards as it breaks the point of the game. Because the will lead to people only playing cards that really fit well, instead of trying to be a bit creative.

File: src/components/views/PlayWhites.js (hunk 1 above)
Comment (anhuwi) [reply src/components/views/PlayWhites.js]: done

File: src/components/views/PlayWhites.js (hunk 2 above)
Comment (anhuwi) [reply src/components/views/PlayWhites.js]: done

File: src/components/views/PlayWhites.js (hunk 3 above)
Comment (anhuwi) [reply src/components/views/PlayWhites.js]: i deleted the skip button

**Review by JoMuelle — APPROVED** [review]

**Pull request merged**